apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php

```diff
@@ -612,7 +612,7 @@ function wpcom_site_has_global_styles_in_personal_plan( $blog_id = 0 ) {
 		return false;
 	}
 
-	$experiment_assignment              = \ExPlat\_load_experiment_and_get_variation( 'calypso_global_styles_personal_v2', $owner, null, false );
+	$experiment_assignment              = \ExPlat\assign_user( 'calypso_global_styles_personal_v2', $owner );
```

Comment:
It would be great if it could be called `assign_given_user` to differentiate it from `assign_current_user` a bit better.

Reply:
FWIW, I personally like the `assign_user` / `assign_current_user` since it matches the format used by Core for other user functions such as [`user_can`](https://developer.wordpress.org/reference/functions/user_can/) / [`current_user_can`](https://developer.wordpress.org/reference/functions/current_user_can/).

Reply:
`assign_user` could also default to the current user if none is given, which is another Core pattern (e.g. [`get_user_option`](https://developer.wordpress.org/reference/functions/get_user_option/), [`is_user_member_of_blog`](https://developer.wordpress.org/reference/functions/is_user_member_of_blog/), etc).

Reply:
Regardless of what we decide to do with the function name, since it's already in use in ETK, to update it we'll need a switcheroo (duplicate the ExPlat function, rename on ETK, delete the old one on ExPlat).

In other words, it doesn't matter if we do it now or while the experiment is live: we can do it at a relaxed pace. 🙂